src/teleop/jetson/arm_trajectory_server.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# Adapted from https://github.com/ros-industrial/robot_movement_interface/blob/master/ur_driver/scripts/joint_trajectory_action.py
+import rospy
+import time
+import actionlib
+import _thread
+from collections import deque
+from typing import Deque
+
+from control_msgs.msg import FollowJointTrajectoryGoal
+from control_msgs.msg import FollowJointTrajectoryResult
+from control_msgs.msg import FollowJointTrajectoryFeedback
+from control_msgs.msg import FollowJointTrajectoryAction
+from trajectory_msgs.msg import JointTrajectory
+from std_msgs.msg import Float64MultiArray
+from sensor_msgs.msg import JointState
+from trajectory_msgs.msg import JointTrajectoryPoint
+
+conf_joint_names = rospy.get_param("teleop/ra_joints/")
+lock = _thread.allocate_lock()
```

Comment:
Would prefer to change to threading.lock()

https://stackoverflow.com/questions/5568555/thread-vs-threading